scripts/generate_json_docs.py

```diff
@@ -295,11 +296,14 @@ def build_type(type_id, title, contents):
 
 
 def clean_source_path(source):
```

Comment:
Right. But what is the input, what needs to be cleaned off of it?

The error reported makes it look like you're using `os.path.abspath(some_module.__file__)` You could just use `some_module.__name__`.